src/Stratis.Bitcoin.IntegrationTests/MinerTests.cs

```diff
@@ -36,6 +41,11 @@ namespace Stratis.Bitcoin.IntegrationTests
 {
     public class MinerTests
     {
+        private const string Password = "password";
+        private const string WalletName = "mywallet";
+        private const string Passphrase = "passphrase";
```

Comment:
This isn't being used.

Reply:
Its used in the test bellow I believe (you don't see the reference?)